src/lib/Hydra/View/NixNAR.pm

```diff
@@ -12,8 +12,9 @@ sub process {
     $c->response->content_type('application/x-nix-archive'); # !!! check MIME type
 
     my $fh = new IO::Handle;
+    my $numThreads = $c->config->{'compress_num_threads'};
```

Comment:
There should also be a default here (`$c->config->{'compress_num_threads'} // 1`), otherwise it won't work for people who don't use `hydra-module.nix`.